helios.js

```diff
@@ -25921,9 +25921,13 @@ export class Address extends CborData {
 	 * Simple payment address without a staking part
 	 * @param {boolean} isTestnet
 	 * @param {PubKeyHash} hash
+	 * @param {?Hash} stakingHash
 	 * @returns {Address}
 	 */
-	static fromPubKeyHash(isTestnet, hash) {
+	static fromPubKeyHash(isTestnet, hash, stakingHash) {
+		if (stakingHash) {
```

Comment:
To stick would the style of the rest of the library:
`if (stakingHash !== null) {return ...} else {return ...}`